mtp_bank_admin/settings/base.py

```diff
@@ -138,6 +138,6 @@
 BANK_STMT_ACCOUNT_NUMBER = os.environ.get('BANK_STMT_ACCOUNT_NUMBER',
                                           '10002383 607080')
 BANK_STMT_CURRENCY = os.environ.get('BANK_STMT_CURRENCY', 'GBP')
-BANK_STMT_OUTPUT_FILENAME = 'stmt_%s.bai'
+BANK_STMT_OUTPUT_FILENAME = 'stmt_%d%m%y.bai'
```

Comment:
I think @madgoose said the BAI files were saved without an extension at SSCL; do we want to match that? (Did I misunderstand?)